target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -425,6 +426,10 @@ ramips_setup_macs()
 		lan_mac=$(mtd_get_mac_ascii factory lanmac)
 		wan_mac=$(mtd_get_mac_ascii factory wanmac)
 		;;
+	dlink,dwr-116-a)
+		lan_mac=$(jboot_config_read -m -i /dev/mtd5 -o 0xE000)
```

Comment:
don't use a fixed mtd path. Something like the following should work:

```
lan_mac=$(jboot_config_read -m -i $(find_mtd_part "config") -o 0xE000)
```

I've seen the same in another file. Please fix it as well.